sdk/storage/storage-blob/test/perfstress/track-1/storageTest.spec.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  Aborter,
+  ContainerURL,
+  ServiceURL,
+  SharedKeyCredential,
+  StorageURL
+} from "@azure/storage-blob";
+import { PerfStressTest } from "@azure/test-utils-perfstress";
+
+// Expects the .env file at the same level as the "test" folder
+import * as dotenv from "dotenv";
+dotenv.config({ path: "../../../.env" });
+
+export abstract class StorageBlobTest<TOptions> extends PerfStressTest<TOptions> {
+  blobServiceClient: ServiceURL;
+  containerClient: ContainerURL;
+  static containerName = `newcontainer${new Date().getTime()}`;
+
+  constructor() {
+    super();
+    const connectionString = StorageBlobTest.getEnvVar("STORAGE_CONNECTION_STRING");
+    const accountName = getValueInConnString(connectionString, "AccountName");
+    const accountKey = getValueInConnString(connectionString, "AccountKey");
+
+    const sharedKeyCredential = new SharedKeyCredential(accountName, accountKey);
+    this.blobServiceClient = new ServiceURL(
+      `https://${accountName}.blob.core.windows.net`,
+      StorageURL.newPipeline(sharedKeyCredential)
+    );
+    this.containerClient = ContainerURL.fromServiceURL(
+      this.blobServiceClient,
+      StorageBlobTest.containerName
+    );
+  }
+
+  public async globalSetup() {
+    await this.containerClient.create(Aborter.none);
```

Comment:
Missing call to super setup and cleanup.

Reply:
This is the base class that extends `PerfStressTest`, in which `globalSetup` and the other methods aren't implemented.
Hence, not required.

Reply:
Doing it anyway for future proofing.